api/v1beta1/solrbackup_types.go

```diff
@@ -64,6 +71,29 @@ func (spec *SolrBackupSpec) withDefaults(backupName string) (changed bool) {
 	return changed
 }
 
+// BackupRecurrence defines the recurrence of the incremental backup
+type BackupRecurrence struct {
+	// Perform a backup on the given schedule, in CRON format.
+	//
+	// Multiple CRON syntaxes are supported
+	//   - Standard CRON (e.g. "CRON_TZ=Asia/Seoul 0 6 * * ?")
+	//   - Predefined Schedules (e.g. "@yearly", "@weekly", "@daily", etc.)
+	//   - Intervals (e.g. "@every 10h30m")
+	//
+	// For more information please check this reference:
+	// https://pkg.go.dev/github.com/robfig/cron/v3?utm_source=godoc#hdr-CRON_Expression_Format
+	Schedule string `json:"schedule"`
+
+	// Define the number of backup points to save for this backup at any given time.
+	// The oldest backups will be deleted if too many exist when a backup is taken.
+	// If not provided, this defaults to 10.
+	//
+	// +kubebuilder:default:=10
+	// +kubebuilder:validation:Minimum:=1
+	// +optional
+	MaxSaved int `json:"maxSaved,omitempty"`
```

Comment:
[0] This is mostly subjective preference, but I'd vote for a smaller default value here.  For collections with a lot of churn, 10 backups might be a lot of data to store by default.  Would you be open to a lower default like 3 or 5?

Reply:
Very open. I literally just chose a number here. I am very happy to differ to whatever you think it should be. Give the call and it is done.

Reply:
I set the default to 5